screens/Home/MyVcs/HistoryTab.tsx

```diff
@@ -16,7 +16,6 @@ export const HistoryTab: React.FC<HistoryTabProps> = props => {
   return (
     <>
       <Modal
-        headerLabel={props.vcMetadata.displayId}
```

Comment:
add header label as fixed string saying `history` so that header doesn't look blank